### PR TITLE
Add linux dependencies to README

### PR DESCRIPTION
**Pull request by yadutaf**
This is most probably non exhaustive but should already help linux users install NextCloud client on their box.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,15 @@ git submodule update --init --recursive
 Run:
 
 ```bash
+# If building on Ubuntu
+sudo apt-get install libsqlite3-dev qt5-default libqt5webkit5-dev qt5keychain-dev
```
Comment (mgerhardy) [inline README.md]: what about libssl1.0-dev?

Comment (yadutaf) [reply README.md]: Just added ``libssl-dev``

**Comment by enoch85** [comment]
@yadutaf Is this really relevant now when you can install PPA on Ubuntu?

**Comment by yadutaf** [comment]
Well, providing accurate build instructions is always a plus for an OpenSource project. Especially one looking for contributors ^^. This said, I'd happily switch to the PPA for my own use. Do you know where to find it ?

**Review by enoch85 — APPROVED** [review]

**Comment by enoch85** [comment]
Ok, lets merge this. 

@yadutaf Read the README.md :)

**Comment by yadutaf** [comment]
> @yadutaf Read the README.md :)

Fair enough ;)

**Pull request merged**